plugins/paymethod/paypal/locale/pt_PT/locale.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/paymethod/paypal/locale/pt_PT/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the pt_BR (Português (Brasil)) locale.
+  -->
+ 
+<locale name="pt_BR" full_name="Português do Brasil">
```

Comment:
Solved aswell, did a `grep -rnw './' -e 'Brasil' | egrep pt_PT`: found two more files
Files:
plugins/importexport/users/locale/pt_PT/locale.xml
plugins/block/developedBy/locale/pt_PT/locale.xml

Also had the same error. br files were my starting point